waiverdb/templates/layout.html

```diff
@@ -17,12 +17,52 @@
         <!-- Bootstrap -->
         <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
         <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-OERcA2EqjJCMA+/3y+gxIOqMEjwtxJY7qPCqsdltbNJuaOe923+mo//f6V8Qbsw3" crossorigin="anonymous"></script>
+        <script src="https://code.jquery.com/jquery-3.7.1.min.js" integrity="sha384-1H217gwSVyLSIfaLxHbE7dRb3v4mYCKbpQvzx0cegeju1MVsGrX5xXxAvs/HgeFs" crossorigin="anonymous"></script>
+        <script>
+const WAIVERS_URL = "{{ url_for('api_v1.waivers_resource') }}";
+
+$(document).ready(function () {
+    $("form").on("submit", function(ev) {
+        ev.preventDefault();
+        let testcaseName = $("#testcase").val();
+        $.ajax({
+            url: WAIVERS_URL,
+            method: "POST",
+            contentType: "application/json",
+            data: JSON.stringify({
+                subject_type: $("#subject_type").val(),
+                subject_identifier: $("#subject_identifier").val(),
+                testcase: testcaseName,
+                product_version: $("#product_version").val(),
+                comment: $("#comment").val(),
+                scenario: $("#scenario").val()
+            }),
+            success: function (data, status, jqXHR) {
+                $("#result-text-error").addClass("d-none");
+                $("#result-text-success").removeClass("d-none");
+                $("#result-link").attr("href", `${WAIVERS_URL}${data.id}`);
+                $("#waiver-result").removeClass("alert-danger d-none").addClass("alert-success");
+            },
+            error: function (jqXHR, textStatus) {
+                $("#result-text-success").addClass("d-none");
+                $("#result-text-error").removeClass("d-none");
+                let addition = "";
+                if(jqXHR.status === 401) {
+                    addition = `<a href="/api/v1.0/permissions?testcase=${testcaseName}&html=1" target="_blank">See who has permissions to waive ${testcaseName}</a>`;
+                }
+                $("#error-desc").html(`Creation failed, status: ${jqXHR.status} ${jqXHR.statusText} ${addition}`);
+                $("#waiver-result").removeClass("alert-success d-none").addClass("alert-danger");
+            }
+        });
+    });
+});
+        </script>
     </head>
 
     <body>
         <div class="container">
             <h1>
-                <img src="{{ url_for('favicon') }}" style="height: 64px" />
+                <img src="{{ url_for('favicon') }}" style="height: 64px" alt="Waiver DB" />
```

Comment:
This is a decorative image, not sure if it deserves alt text.

```suggestion
                <img src="{{ url_for('favicon') }}" style="height: 64px" alt="" />
```

Reply:
'alt' atribute is mandatory for the 'img' tag